include/onnxruntime/core/platform/windows/readme.txt

```diff
@@ -0,0 +1,2 @@
+copied from minkernel/published/internal/telemetry/open_source/TraceLoggingConfig.h 
```

Comment:
Shouldn't this text be in the TraceLoggingConfig.h file? Any reason this should be in a separate readme?